### ISSUE TITLE
Problem in displaying source code with code-tools

### ISSUE THREAD

**Issue by jchiquet** [quarto-dev/quarto-cli#515]
Hi,

I have problems with the use of 'code-tools' and its 'view source' button: the source code is not displayed correctly, remains grayed out and cannot be navigated properly (see for example : https://computo.sfds.asso.fr/published-paper-tsne/ and the source code https://github.com/computorg/published-paper-tsne)

Am I the only one in this case ?

Thank you for you help and for quarto in general!

**Comment by jjallaire** [comment]
@dragonstyle A couple of issues for us to follow up on here:

1) It seems like with recent changes to title block handling the `</> Code` menu is now showing up above the title (rather than at the far right of it

2) I can repro the issues @jchiquet reports (glassed over code-tools) when I set `title-block-banner: true`, e.g. this repros it:

```yaml
---
title: "Untitled"
author: "J.J. Allaire"
title-block-banner: true
format: 
  html:
    code-tools: true
---
```

**Comment by dragonstyle** [comment]
Will do!

**Comment by dragonstyle** [comment]
This should be fixed here: 4ce8bd0808310b95849ea835e452235c369b9d34

New release should be available shortly...

**Comment by jchiquet** [comment]
Indeed, it works with the last release. Thank you!
